src/prototype_room_creepbuilder.js

```diff
@@ -285,7 +288,7 @@ Room.prototype.getPartConfig = function(creep) {
   if (!sufix.fail && !sufix.null) {
     parts = parts.concat(sufix.parts);
   }
-  if (config.debug.spawn) {
+  if (config.debug.spawn && !countOnly) {
```

Comment:
In that case, there will be wall of log messages of this type and that become meaningless, 
I'd rather refactor this function and move log message about spawning in place of real spawning